TestProjects/UniversalGraphicsTest_Lighting/Assets/Test/Editor/UniversalProjectEditorTests.cs

```diff
@@ -0,0 +1,60 @@
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+public class UniversalProjectEditorTests
+{
+    private static UniversalRenderPipelineAsset currentAsset;
+
+    [Test]
+    public void GetCurrentAsset()
+    {
+        GetUniversalAsset();
+    }
+
+    [Test]
+    public void CheckAllLightingSettings()
```

Comment:
Why duplicate? It is available here com.unity.testing.urp/Scripts/Editor/UniversalProjectEditorTests.cs

Reply:
This was some refactoring done by URP team and then it was reverted. The revert PR is merged to my branch to make things work again:
https://github.com/Unity-Technologies/Graphics/pull/4257/commits/5b8fb9220633fffdfcf9189fbec2eb645fe6508c